src/components/modal/ModalHeader.ts

```diff
@@ -23,14 +23,19 @@ export const ModalHeader = styled(Box)<{ hasBorder?: true }>`
 `;
 
 export const ModalTitle = styled(Typography)`
-  font-size: 24px;
-  line-height: 36px;
+  font-size: 20px;
+  line-height: 30px;
   font-weight: 600;
   color: black;
-  position: absolute;
-  left: 0;
-  right: 0;
-  margin-left: auto;
-  margin-right: auto;
   text-align: center;
+
+  @media screen and (min-width: 1024px) {
+    font-size: 24px;
+    line-height: 36px;
+    position: absolute;
+    left: 0;
+    right: 0;
+    margin-left: auto;
+    margin-right: auto;
```

Comment:
Non serve il responsive. Lo si usa con isMobile nel modal Header e nel corpo del Modal si usa un Typography provenienta da MUI. Guarda il la PR del `SucessModal`

Reply:
Si, era per utilizzare lo stesso componente già pronto tutte le volte e non dover ogni volta specificare gli stessi valori al Typography di mui

Reply:
Non specificare i valori a MUI tutte le volte per i testi può essere una feature successiva in cui andiamo a fare per la Typography lo stesso lavoro che abbiamo fatto per i componenti del Modal. 

Però questa non è una soluzione valida, inizializza due volte lo stesso componente che ha del responsive all'intero, nonostante usi anche un isMobile.